src/router.js

```diff
@@ -71,7 +71,7 @@ router.afterEach(unlessFailure(to => {
       // Implements the restoreSession meta field. A test can skip this request
       // by setting the session before the initial navigation.
       async (to) => {
-        if (to.meta.restoreSession && !session.dataExists) {
+        if (/*to.meta.restoreSession &&*/ !session.dataExists) {
```

Comment:
I don't understand this line, is this change just for development purpose?

Reply:
I'll add a comment.